test/cat.jl

```diff
@@ -55,13 +47,29 @@ module TestCat
     end
 
     @testset "hcat ::Vectors" begin
+        df = DataFrame()
+        DataFrames.hcat!(df, CategoricalVector{Union{Int, Missing}}(1:10))
```

Comment:
Am I missing something, or this duplicated below?

Reply:
It was old code that I simply moved. I have cleaned it up now.